bridge/github/config.go

```diff
@@ -205,13 +205,42 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
```

Comment:
Could you try to match the behavior I detailed in https://github.com/MichaelMure/git-bug/issues/93#issue-401117315 ?

In particular, the user should be able to select an option with just the number, and the `[0]` should allow to enter an arbitrary project, as the code does before your PR.

Reply:
It's not important how much code we need to write. What is important is to build a tool that is fast, easy to use and understand.

Reply:
Okay wonderful I will map the index values to the URL and submit a PR.